packages/app-desktop/gui/NoteList/style.scss

```diff
@@ -36,6 +36,10 @@
 		width: 2px;
 		height: 2px;
 	}
+
+	&:focus {
+		outline: none;
+	}
```

Comment:
If removed, the focus indicator in the note list looks like this, and is shown when pressing the up/down arrow keys:
<img alt="orange outline, partially clipped by other boxes" src="https://github.com/laurent22/joplin/assets/46334387/8ea3c920-9a40-45d4-9929-b38600804692" width="250"/>

Here's an alternative:
```suggestion
	&:focus-visible {
		outline: none;

		> .note-list-item > .content {
			background-color: var(--joplin-background-color-hover3);
		}
	}
```

While the above SCSS works, it also breaks RCSS design guidelines. It produces the following:
<img alt="Note list. Current item has a light blue background" width="250" src="https://github.com/laurent22/joplin/assets/46334387/0654a4cc-026c-4dc1-8d99-b15c4c845c77"/>

Reply:
That's fine if there's no other way

Reply:
This has been resolved by changing the default list renderer.